source/index.html.md.erb

```diff
@@ -1,5 +1,5 @@
 ---
-title: CheddarGetter API Reference
+title: Cheddar API Reference
```

Comment:
Right. There's a [closed Github issue at Middleman](https://github.com/middleman/middleman/issues/1246) about this. I could change anything that uses the title front matter variable to instead rely on the global config variable. However, I suspect that doing so could increase the number of merge conflicts the next time we upgrade Slate.